plan/stats.go

```diff
@@ -35,7 +35,7 @@ func (s *statsInfo) Count() int64 {
 	return int64(s.count)
 }
 
-// collapse receives a selectivity and multiple it with count and cardinality.
+// scale receives a selectivity and multiple it with count and cardinality.
```

Comment:
The word "multiple" can be used as a noun or an adjective, but not a verb. Perhaps you mean "multiply" here?

Reply:
1. Yes, Here needs a verb.
2. We need the `selectivity` here.